supabase/functions/binance-connector/index.ts

```
@@ -1,320 +1,98 @@
+          reduceOnly,
+          positionSide: body.positionSide,
+        });
+        return jsonResponse({ success: true, order });
       }
+        const order = await connector.placeOrder({
+          symbol: body.symbol,
+          side: body.side,
+          type: body.type,
+          quantity: Number(body.quantity),
+          price: body.price !== undefined ? Number(body.price) : undefined,
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)